README.rst

```diff
@@ -7,23 +7,23 @@ work with Open Dylan and ``c-ffi``.
 Build
 -----
 
-Be sure that you cloned this repository recursively or have otherwise
-initialized and updated the git submodules. This will not build without
-the submodules present.
+Be sure that you cloned this repository recursively and have run ``deft update`` to
```

Comment:
Is still needed to clone the repository recursively if the submodules are going to be deleted?

Reply:
Fixed, thanks.